services/supabase/migrations/20260827000002_create_story_images_bucket.sql

```
@@ -0,0 +1,29 @@
+-- Create a public Supabase Storage bucket for project story images.
+-- Images are served publicly via their hosted URL so they can be embedded
+-- in story HTML without authentication.
+
+INSERT INTO storage.buckets (id, name, public)
+VALUES ('project_story_images', 'project_story_images', true)
+ON CONFLICT (id) DO NOTHING;
+
+-- Allow authenticated users with project-management permissions to upload.
+-- Fine-grained permission enforcement is done at the API layer (story-image route).
+CREATE POLICY "Authenticated users can upload story images"
```

SQLFluff (4.3.0)

[error] Do not use special characters in identifiers.

(RF05)